pkg/package.go

```diff
@@ -161,3 +161,36 @@ func (p Package) MarshalText() string {
 func (p Package) String() string {
 	return fmt.Sprintf("package '%s'", p.PID.String())
 }
+
+func (p Package) Create() error {
+	state, err := p.State()
+	if err != nil {
+		return err
+	}
+	if state.Exists {
+		return fmt.Errorf("%s > package '%s' cannot be create as it exists", p.manager.instance.ID(), p.PID.String())
+	}
+	return p.manager.Create(state.PID)
+}
+
+func (p Package) Update(filter []Filter) error {
```

Comment:
This is more like UpdateFilters method. In go there is no method overloading so the names need to be different